### ISSUE TITLE
Saga effects should be yielded

### ISSUE THREAD

**Issue by gihrig** [mxstbr/react-boilerplate#1232]
After dependency update #1225 many new style errors are flagged. I worked around this one by rolling back the dependency to the earlier version, but IMO, the code should be corrected and the newer version of eslint-plugin-redux-saga installed.

To reproduce:

1. `npm install --save-dev eslint-plugin-redux-saga@0.2.0`
2. `npm run lint`

Error:
```
~/react-bp/app/containers/HomePage/tests/sagas.test.js
39:35 error put effect must be yielded redux-saga/yield-effects
...
54:36 error fork effect must be yielded redux-saga/yield-effects
...
70:42 error take effect must be yielded redux-saga/yield-effects
```
_eslint-plugin-redux-saga 0.1.5 → 0.2.0 <--- rolled back_

see: https://github.com/pke/eslint-plugin-redux-saga/blob/develop/docs/rules/yield-effects.md

**Comment by gihrig** [comment]
Well, I took a crack at this. Clearly I'm not well versed in the area of sagas, much less sagas and testing.

The following error demonstrates the recommended [solution](https://github.com/pke/eslint-plugin-redux-saga/blob/develop/docs/rules/yield-effects.md) and its failure. Note that this solution is also what is implemented automatically when `--fix` is added to the eslint command line.

It's interesting to note that the `redux-saga` project apparently disregards this rule for internal project [testing](https://github.com/yelouafi/redux-saga/blob/master/examples/counter/test/sagas.js#L16-L20).

![image](https://cloud.githubusercontent.com/assets/1481063/20863927/aebb5684-b991-11e6-94a5-da3a30f06727.png)

From here my "solution" would be to simply disable this rule in the offending test. 

`/* eslint-disable redux-saga/yield-effects */`

I would also suggest disabling this rule in [`internals/generators/container/sagas.test.js.hbs`](https://github.com/mxstbr/react-boilerplate/blob/master/internals/generators/container/sagas.test.js.hbs)

In the absence of more constructive input, I expect to implement this fix in the next dependency update (at least a week from now).

Thoughts, @anyone?

**Comment by gihrig** [comment]
Resolved by #1328
